hazelcast/src/test/java/com/hazelcast/test/HazelcastParallelClassRunner.java

```diff
@@ -59,6 +60,8 @@ private static int getDefaultMaxThreads() {
             // on parallelism inside each JVM. otherwise it's easy to use too much resource
             // and the test duration is actually longer and not shorter.
             cpuWorkers = min(4, cpuWorkers);
+            // decrease the amount of resources used when running in multiple JVM
+            RuntimeAvailableProcessors.override(8);
```

Comment:
There are several calls to `RuntimeAvailableProcessors#resetOverride` from tests, this resets the available processors back to the `Runtime.getRuntime().availableProcessors()`. It seems like we need a stronger override here :)

Reply:
Good call :)